### DISCUSSION TITLE
PF2E Date alignment

### DISCUSSION THREAD

**Discussion by bucket-au** [vigoren/foundryvtt-simple-calendar#539] [Q&A]
Hi, having trouble with something. Using PF2E, but the dates in Simple Calendar don't align with what is coming up in chat.

![image](https://github.com/vigoren/foundryvtt-simple-calendar/assets/10415210/d800758d-ae5a-4185-bcf4-2590c9d54a73)

So, Simple Calendar says the year is 4721, but the game system says 2698. Even though I change the settings for the date in Simple Calendar?

**Comment by vigoren** [comment]
Hey, this is a bug specific to PF2E. I should have a fix for it later tonight.

**Reply by vigoren** [reply]
This has been fixed in version 2.4.0
